README.md

```diff
@@ -73,27 +73,36 @@ pip3 install python-nubia
 
 ## Building Nubia from source
 
-Ensure is pipenv installed:
-```bash
-pip3 install pipenv
-```
 You can either setup.py to build a tarball, or use pipenv to setup a virtualenv with all the dependencies installed.
 
 ## Running example in virtualenv:
 
+It's often best to create a virtualenv to contain the dependencies required for python-nubia project.
+```bash
+ python3 -m virtualenv .venv
```

Comment:
Please remove the leading whitespace.
Also, shouldn't it be `venv` module? I get `No module named virtualenv` error when following these instructions.

Reply:
Checked this on Python 3.6 an 3.7, it should be `venv`.

Reply:
Fixed.